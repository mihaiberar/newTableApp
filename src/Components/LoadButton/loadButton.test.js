import { LoadButton } from './loadButton.jsx'
import { render } from '@testing-library/react'

describe('<LoadButton> Snapshot', () => {
  it('should capture Snapshot of <LoadButton/> component', () => {
    const { container } = render(<LoadButton />)
    expect(container).toMatchSnapshot()
  })
})

it('should render with a className equal to the variant', () => {
  const { container } = render(<LoadButton />)
  expect(container.firstChild.classList.contains('loadBtn')).toBe(true)
})
