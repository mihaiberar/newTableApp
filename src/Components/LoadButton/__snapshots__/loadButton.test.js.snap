// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadButton> Snapshot should capture Snapshot of <LoadButton/> component 1`] = `
<div>
  <button
    class="loadBtn"
  />
</div>
`;
